JEasyCryptoClient/build.sh

```diff
@@ -1 +1,2 @@
+mkdir bin
```

Comment:
But what if the directory already exists? Than the script breaks... It would be better if the script could manage existing folders.

Reply:
Thank you for the comment. I will make changes and get back to you.

Reply:
Hello, I changed the mkdir commands to include the -p flag, which creates the folder if needed. Is this acceptable, or should there be a more in-depth check?

Reply:
This is totally acceptable! Thank you.